pilot/cmd/pilot-agent/options/agent.go

```diff
@@ -0,0 +1,62 @@
+// Copyright Istio Authors
```

Comment:
This should probably not be under pilot/cmd - maybe in pkg/isito-agent ?

Reply:
Personally I donot like to move options under pkg. I'd like to move it and also the main under istio-agent/xxx .